dimod/higherorder/polynomial.py

```diff
@@ -127,10 +134,10 @@ def __eq__(self, other):
             except Exception:
                 # not a polynomial
                 return False
-
+        
```

Comment:
not sure why these lines needed changes?

Reply:
Looks like an eccidental whitespace change. Will fix. I am not seeing any semantic changes

Reply:
Looks like an eccidental whitespace change. Will fix. I am not seeing any semantic changes

Reply:
Do we normally have whitespace in blank lines up to indentation level? Seems like that is what I removed

Reply:
No, in fact removing the whitespace is correct. It's more about keeping the blame/commit history clean. No big deal either way.